autoload/crystal_lang.vim

```diff
@@ -51,10 +51,17 @@ function! crystal_lang#entrypoint_for(file_path) abort
         return a:file_path
     endif
 
+    let spec_for_file = get(b:, 'crystal_spec_file', get(g:, 'crystal_spec_file', ''))
```

Comment:
I prefer more descriptive variable name. How about `crystal_required_spec_path`? To clarify it represents files to run specific spec, I feel adding `required` would be obvious. And the value is actually (globbed) path. So I prefer 'path' to 'file'.